lib/judge/controller.rb

```diff
@@ -9,11 +9,11 @@ def self.included(base)
     end
 
     def validation(params)
-      params = normalized_params(params)
-      if params_present?(params) && params_exposed?(params)
-        Validation.new(params)
+      norm_params = normalized_params(params)
```

Comment:
@aeberlin if it's relevant to the PR, would you mind putting the examples on the PR... I'd love to have that as a reference & record in the future